Docs-css/Docs-layout.css

```diff
@@ -0,0 +1,27 @@
+.inner-container {
+  height: auto;
+  display: flex;
+  justify-content: space-around;
+  background-color: var(--bg-clr);
+}
+
+.customThemeBox {
+  box-shadow: 0 3px 13px -6px #ccc, 3px 0 10px -6px #ccc;
+  border-radius: 8px;
+}
+
+.code-container{
+  width: 96%;
+  margin: 3rem 0 !important; 
```

Comment:
Can we try to avoid !important, instead put this class below the one whose style is getting applied when you dont use !important

Reply:
I tried , the style class is coming from prism.css file , i tried putting my stylesheet link below the prism.css file , but didnt worked thats why i have to use that

Reply:
Then the style must be getting applied from somewhere else as well, inspect that particular component and see which one is overwriting. 
Or can we try to add this class at the end of prism.css file and test

Reply:
Done , changed the code in prism.css, now its working , thanks for such great review

Reply:
Amazing